conf/repos.json

```diff
@@ -1,14 +1,16 @@
 {
     "repos": [
-     {
-          "name": "DataStream",
-          "url": "https://api.datacite.org/dois",
-          "homepage_url": "https://datastream.org/",
-          "thumbnail": "https://test.frdr.ca/discover/img/sources/datastream_80x80.png",
-          "type": "datastream",
-          "set": "wdgcf.datastream",
-          "item_url_pattern": "https://datastream.org/dataset/%id%",
-          "enabled": true
-      }
+        {
+         "name": "FRDR",
+         "url": "https://dev6.frdr.ca/oai/request",
```

Comment:
Good to add this file to .gitignore, but probably should push a better version of it as well, one that doesn't have dev6 in the url.